examples/responsive-vis/responsive-vis-utils.js

```diff
@@ -0,0 +1,122 @@
+import {randomNormal} from 'd3-random';
+import {scaleLinear, scaleQuantize, scaleSqrt} from 'd3-scale';
+import {max, range} from 'd3-array';
+
+// create fake data
+export function createData(dataSize, barChart = true) {
```

Comment:
generally, I prefer using the const arrow function form to be sure they can't be overridden.